src/pages/CalendarPage.js

```diff
@@ -34,7 +33,7 @@ export default function CalendarPage() {
     <>
       <Heading text={'Du bist ein Kalender'} />
       {data.map((object, index) => (
-        <Dayentry key={index} text={object.text} datum={object.date} />
+        <JournalEntry key={index} text={object.text} datum={object.date} />
```

Comment:
I would not recommand the index as an key, since your data might change later during development (manybe you will implement delete functions later) . For now its fine, but consider unique ids later on :)